examples/nas/enas/search.py

```diff
@@ -43,5 +43,6 @@
                                num_epochs=num_epochs,
                                dataset_train=dataset_train,
                                dataset_valid=dataset_valid,
-                               log_frequency=args.log_frequency)
-    trainer.train_and_validate()
+                               log_frequency=args.log_frequency,
+                               controller=controller)
```

Comment:
is there default value for controller?

Reply:
Yes. None.

Reply:
does this trainer work with None? if not, please specify a default controller

Reply:
Yes. See line 24.

Reply:
ok